src/core/turn-order.js

```diff
@@ -44,8 +44,11 @@ export const TurnOrder = {
    * The default round-robin turn order.
    */
   DEFAULT: {
-    first: (G, ctx) => ctx.currentPlayer,
-    next: (G, ctx) => (+ctx.currentPlayer + 1) % ctx.numPlayers + '',
+    first: (G, ctx) => ctx.playerOrder[0] + '',
+    next: (G, ctx) => {
+      let playerPos = ctx.playerOrder.findIndex(x => x == ctx.currentPlayer);
```

Comment:
This will break if we want to support duplicate entries in `playerOrder`. Instead, why don't we store `playerOrder` and `_playerOrderIndex` in `ctx`? `_playerOrderIndex` is just an integer that's incremented normally that's an index into the `playerOrder` array.

Also, I have a mild preference to name it `playOrder` rather than `playerOrder`.

Reply:
@nicolodavis OK. Is there any particular reason to stick with currentPlayer as string instead of an integer?

Reply:
So that we can change to actual string ID's (like "Bob" or "Alice") in the future easily.